README.md

```diff
@@ -18,7 +18,7 @@ a change is made.
 How to use
 ----------
 
-### Travis
+### Travis (with conda)
```

Comment:
maybe link to the section without conda in the first sentence after this?

Reply:
Sounds good will add

Reply:
@astrofrog , after adding it, I think we can merge? Also if you really like green status, rebase and you shall have it.

Reply:
Done! Because master is a protected branch we do need to wait for the CI to run again though.